### PR TITLE
[13.0][PORT] 561 from 10.0

### PR DESCRIPTION
**Pull request by sebastienbeau**
Port #561 from 10.0 to 13.0.

### CODE REVIEW

**Comment by sebastienbeau** [comment]
Forward port fail as this PR already add the field https://github.com/shopinvader/odoo-shopinvader/pull/734
TODO we need to check if we do not miss some feature between the two implementation

**Review by PierrickBrun — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

File: shopinvader/services/address.py [hunk 1]
```diff
@@ -71,15 +71,22 @@ def _store_cache_needed(self, partner):
...
         return validator
 
     def _validator_create(self):
         res = {
-            "type": {"type": "string", "default": "other"},
             "name": {"type": "string", "required": True},
+            "type": {
+                "type": "string",
+                "allowed": self._get_allowed_type(),
+                "default": "other",
```
Comment (simahawk) [inline shopinvader/services/address.py]: maybe "contact" is better as default?

Comment (PierrickBrun) [reply shopinvader/services/address.py]: absolutely

**Review by PierrickBrun — APPROVED** [review]

**Pull request merged**